pkg/navigation/client/publishing.go

```diff
@@ -16,7 +16,7 @@ type PublishingClient struct {
 	languages       []string
 }
 
-func NewPublishingClient(ctx context.Context, clients *Clients, languages []string) Clienter {
+func NewPublishingClient(_ context.Context, clients *Clients, languages []string) Clienter {
```

Comment:
This is a public function and if `ctx` isn't needed it should be removed entirely. However, that would be a larger refactor and a major bump in versioning. I think you should "ignore" it in this instance